.gitignore

```diff
@@ -13,3 +13,4 @@ stdx-allocator-test-*
 *.o
 *.obj
 *.lst
+dub.selections.json
```

Comment:
I am not sure. I thought this file is for applications or production libraries. At lest, it will prevent travis to test with the latest mir-core patch